vrc-get-gui/app/settings/page.tsx

```diff
@@ -23,6 +28,7 @@ import i18next, {languages, tc, tt} from "@/lib/i18n";
 import {VGOption, VGSelect} from "@/components/select";
 import {useFilePickerFunction} from "@/lib/use-file-picker-dialog";
 import {emit} from "@tauri-apps/api/event";
+import {appWindow} from "@tauri-apps/api/window";
```

Comment:
due to next.js tauri limitations, api/window is not possible to import so please dynamically import appWindow inside changeTheme please.